controller_manager/doc/userdoc.rst

```diff
@@ -177,23 +176,25 @@ There are two scripts to interact with controller manager from launch files:
 .. code-block:: console
 
     $ ros2 run controller_manager unspawner -h
-    usage: unspawner [-h] [-c CONTROLLER_MANAGER] controller_name
+    usage: unspawner [-h] [-c CONTROLLER_MANAGER] [--controller-manager-switch-timeout CONTROLLER_MANAGER_SWITCH_TIMEOUT] controller_names [controller_names ...]
 
     positional arguments:
-      controller_name       Name of the controller
+      controller_names      Name of the controller
 
-    optional arguments:
+    options:
       -h, --help            show this help message and exit
       -c CONTROLLER_MANAGER, --controller-manager CONTROLLER_MANAGER
                             Name of the controller manager ROS node
+      --controller-manager-switch-timeout CONTROLLER_MANAGER_SWITCH_TIMEOUT
+                            Time to wait for the controller manager to switch controllers
 
 ``hardware_spawner``
 ^^^^^^^^^^^^^^^^^^^^^^
 
 .. code-block:: console
 
     $ ros2 run controller_manager hardware_spawner -h
-    usage: hardware_spawner [-h] [-c CONTROLLER_MANAGER] (--activate | --configure) hardware_component_name
+    usage: hardware_spawner [-h] [-c CONTROLLER_MANAGER] [--controller-manager-timeout CONTROLLER_MANAGER_TIMEOUT] (--activate | --configure) hardware_component_name
```

Comment:
Do we have it for the hardware spawner as well?

Reply:
the timeout for this service call yes

https://github.com/ros-controls/ros2_control/blob/0433960580c5834b11af9703e3e34ace86824d01/controller_manager/controller_manager/hardware_spawner.py#L159-L160

Reply:
Ahh ok ok, this is a different one. LGTM